src/webgpu/api/validation/render_pass/resolve.spec.ts

```diff
@@ -0,0 +1,120 @@
+export const description = `API Validation Tests for RenderPass Resolve.
+
+  Test Coverage:
+    - When resolveTarget is not null:
+      - Test that the colorAttachment is multisampled:
+        - A single sampled colorAttachment should generate an error.
+      - Test that the resolveTarget is multisampled:
+        - A multisampled resolveTarget should generate an error.
+      - Test that the resolveTarget has usage OUTPUT_ATTACHMENT:
+        - A resolveTarget without usage OUTPUT_ATTACHMENT should generate an error.
+      - Test that the resolveTarget's texture view describes a single subresource:
+        - An error should be generated when:
+          - The resolveTarget view mip count is not 1.
+          TODO: The resolveTarget view layer count is not 1.
+          TODO: The resolveTarget view slice count is not 1.
+      - Test that the resolveTarget's format is the same as the colorAttachment:
+        - An error should be generated when the resolveTarget's format does not match the
+          colorAttachment's format.`;
```

Comment:
The resolve target should be the same size as the attachment.

Reply:
Added.